modules/ROOT/pages/monitoring_settings_page.adoc

```diff
@@ -0,0 +1,34 @@
+= Anypoint Monitoring Settings
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+On the Settings page, you can:
+
+* Find the instructions to install Anypoint Monitoring on an on-premises server to monitor applications running on that server. 
```

Comment:
Do they only monitor Mule applications or all apps on the server? (don't need to say if context is set already).